R/AlgorithmHandler.r

```diff
@@ -0,0 +1,143 @@
+library("base64enc")
+library("rjson")
+
+
+
+getResponseObject_ <- function(output) {
+    if (typeof(output) == "raw") {
+      list(
+        result = base64enc::base64encode(output),
+        metadata = list(content_type = "binary")
+      )
+    } else if (is.character(output) &
+               length(output) == 1) {
+      list(result = output,
+           metadata = list(content_type = "text"))
+    } else {
+      list(result = output,
+           metadata = list(content_type = "json"))
+    }
+  }
+getResponseAsJsonString_ <- function(output) {
+    tryCatch({
+      rjson::toJSON(output)
+    },
+    error = function(e) {
+      print(paste0("Error in getResponse: ", e))
+      rjson::toJSON(list(
+        error = list(
+          message = toString(e),
+          stacktrace =
```

Comment:
the stacktrace thing can all be on one line. And let's indent this to match the indentation of the "error" line